KenticoCloud.Delivery/DeliveryClient.cs

```diff
@@ -272,10 +256,8 @@ public async Task<DeliveryItemResponse<T>> GetItemAsync<T>(string codename, IEnu
         /// </summary>
         /// <param name="parameters">An array that contains zero or more query parameters, for example, for filtering, ordering, or setting the depth of modular content.</param>
         /// <returns>The <see cref="DeliveryItemListingResponse"/> instance that contains the content items. If no query parameters are specified, all content items are returned.</returns>
-        public async Task<DeliveryItemListingResponse> GetItemsAsync(params IQueryParameter[] parameters)
-        {
-            return await GetItemsAsync((IEnumerable<IQueryParameter>)parameters);
-        }
+        public async Task<DeliveryItemListingResponse> GetItemsAsync(params IQueryParameter[] parameters) => 
+            await GetItemsAsync((IEnumerable<IQueryParameter>)parameters);
```

Comment:
Would you mind moving the "arrow" on the line with expression? I know it is a tiny detail and a matter of habit but since we write symbols of the ternary operator and dots in longer expressions at the beginnings of lines, it seems a bit more natural to write the arrow on the new line as well to me.